### PR TITLE
🥵Update Homepage

### PR DESCRIPTION
**Pull request by ashwin1104**
Update baseline design for homepage
Include header menu icons, video player, home block, card hover, and world map

### CODE REVIEW

File: client/public/style.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  margin: 1.5px;
+}
+.sdg-row {
+  margin-top: 50px;
+}
+.sdg-col {
+  padding: 0px;
+}
+
+// .container {
```
Comment (arpanlaha) [inline client/public/style.scss]: can we remove the commented out stuff?

File: client/pages/index.jsx [hunk 2]
```diff
@@ -1,13 +1,131 @@
...
+    return (
+      <Container>
+        <Row className="header-row" xs="15" justify="center" align="middle">
+          <Col>
+            <img className="header-img" src="/homepage-images/menu-icon.png" />
+          </Col>
+          <Col>
+            <img className="header-img" src="/homepage-images/kso-icon.png" />
+          </Col>
+          <Col></Col>
```
Comment (arpanlaha) [inline client/pages/index.jsx]: are all these necessary?

File: client/pages/index.jsx [hunk 3]
```diff
@@ -1,13 +1,141 @@
...
+            <Col>
+              <img
+                className="header-img"
+                src="/homepage-images/menu-icon.png"
+              />
+            </Col>
+            <Col>
+              <img className="header-img" src="/homepage-images/kso-icon.png" />
+            </Col>
+            <Col></Col>
```
Comment (arpanlaha) [inline client/pages/index.jsx]: are all these necessary?

File: client/pages/index.jsx [hunk 4]
```diff
@@ -1,13 +1,121 @@
...
+          <Col className="home-block-col">
+            <a className="text-link" href="about us">
+              ABOUT US
+            </a>
+            <h1 className="header-text">Change the World,</h1>
+            <h1 className="header-text" mode="single">
+              one step at a time.
+            </h1>
+            <p className="body-text">
+              The barred ratio flips under the proved nail. An umbrella brushes
```
Comment (arpanlaha) [inline client/pages/index.jsx]: where is this from lol

File: client/pages/index.jsx [hunk 5]
```diff
@@ -1,13 +1,121 @@
...
+        <Row className="sdg-row">
+          {this.sdgImages.map(sdgImage => (
+            <Col key={sdgImage.id} className="sdg-col" sm="2">
+              <CardGroup>
+                <Card
+                  className="sdg-card"
+                  tag="a"
+                  onClick={() => console.log("clicked")}
+                  style={{
+                    cursor: "pointer",
```
Comment (arpanlaha) [inline client/pages/index.jsx]: we should avoid inline styling unless necessary

File: client/pages/_app.jsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (arpanlaha) [inline client/pages/_app.jsx]: we can delete this file it isn't needed anymore

File: client/pages/index.jsx [hunk 7]
```diff
@@ -1,13 +1,121 @@
...
+      id: 18,
+      imageLink: `/sdg-images/sdg.png`
+    };
+    sdgImages.push(sdgObject);
+  };
+
+  render() {
+    this.populateImagesArr(this.sdgImages);
+    return (
+      <Container>
```
Comment (arpanlaha) [inline client/pages/index.jsx]: enclose all of this in a jsx fragment and put `<Head />` above container

File: client/pages/index.jsx (hunk 4 above)
Comment (ashwin1104) [reply client/pages/index.jsx]: It's filler text it was a random paragraph generator

File: client/pages/index.jsx [hunk 8]
```diff
@@ -1,13 +1,118 @@
...
+  Col,
+  Container,
+  Input,
+  Row
+} from "reactstrap";
+
+import "../public/style.scss";
+
+const populateImages = () => {
+  const images = new Array(19);
```
Comment (yousefa00) [inline client/pages/index.jsx]: Magic numbers, I feel like a 126 code mod lol im sorry

File: client/pages/index.jsx [hunk 9]
```diff
@@ -1,13 +1,118 @@
...
+
+  for (let i = 1; i <= 17; i++) {
```
Comment (yousefa00) [inline client/pages/index.jsx]: same here

File: client/pages/index.jsx (hunk 8 above)
Comment (arpanlaha) [reply client/pages/index.jsx]: oops i did this one one sec

**Pull request merged**